mywebsite/news/templates/news/base.html

```diff
@@ -68,6 +68,12 @@ <h1>Footer</h1>
            href="{% url 'news:article_feed' %}">
             <i class="fa fa-rss" aria-hidden="true">RSS</i>
         </a>
+    </div>
+        <div class="container">
+        <a data-mdb-ripple-init class="btn btn-primary" role="button" data-mdb-ripple-color="dark"
+           href="{% url 'news:django.contrib.sitemaps.views.sitemap' %}">
```

Comment:
к этому относится то что ниже про файл mywebsite/news/urls.py
```suggestion
           href="{% url 'news:слон' %}">
```